object_service/tests/test_internals.py

```diff
@@ -36,7 +41,7 @@ def test_config_values(self):
                     "OBJECTS_SOLRQUERY_URL",
                     "API_URL"
                     ]
-        missing = [x for x in required if x not in self.app.config.keys()]
+        missing = [x for x in required if x not in list(self.app.config.keys())]
```

Comment:
I think this `list(...)` (automatically added by the porting tools) might not be needed here.